Source/WebCore/platform/network/create-http-header-name-table

```diff
@@ -137,21 +137,20 @@ bool findHTTPHeaderName(StringView stringView, HTTPHeaderName& headerName)
         return false;
 
     if (stringView.is8Bit()) {
-        if (auto nameAndString = HTTPHeaderNamesHash::findHeaderNameImpl(reinterpret_cast<const char*>(stringView.characters8()), length)) {
+        if (auto nameAndString = HTTPHeaderNamesHash::findHeaderNameImpl(reinterpret_cast<const char*>(stringView.span8().data()), length)) {
             headerName = nameAndString->headerName;
             return true;
         }
     } else {
-        LChar characters[maxHTTPHeaderNameLength];
-        for (unsigned i = 0; i < length; ++i) {
-            UChar character = stringView.characters16()[i];
+        char characters[maxHTTPHeaderNameLength];
+        size_t index = 0;
+        for (auto character : stringView.span16()) {
             if (!isASCII(character))
```

Comment:
Seems like we could do the `containsOnlyASCII` check separately and then not have to write a loop just to put the characters into an array. This can’t be the only place that wants to get a `StringView` into a character array; I suppose it’s optimized to only check for ASCII in the 16-bit case.